plugins/modules/mso_schema_template_vrf.py

```diff
@@ -46,6 +47,15 @@
     description:
     - Whether to enable vzAny.
     type: bool
+  ip_data_plane_learning:
+    description:
+    - Whether IP data plane learning is enabled or disabled.
+    type: str
```

Comment:
should we use boolean type, and then map to enabled/disabled?

Reply:
will keep it as string for now.